src/ResourceManager/RecoveryServices.Backup/Cmdlets/Container/GetAzureRmRecoveryServicesContainer.cs

```diff
@@ -54,34 +50,30 @@ public class GetAzureRmRecoveryServicesContainer : RecoveryServicesBackupCmdletB
         public override void ExecuteCmdlet()
         {
             ExecutionBlock(() =>
-                {
-                    base.ExecuteCmdlet();
-
-                    ProtectionContainerListQueryParams queryParams = new ProtectionContainerListQueryParams();
-
-                    // 1. Filter by Name
-                    queryParams.FriendlyName = Name;
+            {
+                base.ExecuteCmdlet();
 
-                    // 2. Filter by ContainerType
-                    queryParams.ProviderType = HydraHelpers.GetHydraProviderType(ContainerType);
+                PsBackupProviderManager providerManager = new PsBackupProviderManager(new Dictionary<System.Enum, object>()
+                {  
+                    {ContainerParams.Vault, Vault},
+                    {ContainerParams.ContainerType, ContainerType},
+                    {ContainerParams.Name, Name},
+                    {ContainerParams.ResourceGroupName, ResourceGroupName},
```

Comment:
Can you confirm with PM for param name whether it should be ResourceGroupName or VMResourceGroupName?